components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/impl/EndpointCertificatesApiServiceImpl.java

```diff
@@ -226,6 +230,33 @@ public Response updateEndpointCertificateByAlias(String alias, InputStream certi
         return null;
     }
 
+    @Override
+    public Response getCertificateUsageByAlias(String alias, Integer limit, Integer offset, MessageContext messageContext) throws APIManagementException {
+
+        limit = (limit != null) ? limit : RestApiConstants.PAGINATION_LIMIT_DEFAULT;
+        offset = (offset != null) ? offset : RestApiConstants.PAGINATION_OFFSET_DEFAULT;
+
+        APIMetadataListDTO apiMetadataListDTO;
+        CertificateMetadataDTO certificateMetadataDTO;
+        String fqdn;
+        APISearchResult searchResult;
+        String organization = RestApiUtil.getValidatedOrganization(messageContext);
+        APIProvider apiProvider = RestApiCommonUtil.getLoggedInUserProvider();
+        certificateMetadataDTO = apiProvider.getCertificate(alias);
+
+        if(certificateMetadataDTO != null) {
```

Comment:
Shall we fix this formatting issue? A white space is required.

Reply:
Fixed https://github.com/wso2/carbon-apimgt/pull/11739